### PR TITLE
Rfp/security marketplace

### PR DESCRIPTION
**Pull request by ParthChaudhary31**
#### Project Abstract

> Please replace these instructions with a brief description of your project summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [x] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by ParthChaudhary31** [comment]
> Thanks @ParthChaudhary31 you mentioned that this would help save on development costs, does this mean there should be an adjustment in price as well? Or are these only contract deployment costs that aren't covered as part of the grant?

The latter actually, The cost-cutting would be on the front of :
#1) Deployment costs (Contract optimization by merging functionality into a single contract would require deploying fewer contracts).
#2) The Arbiters will not need to spend from their own pockets while voting. (Minimising gas fees Charges).

**Review by keeganquigley — APPROVED** [review]
Thanks for clarifying @ParthChaudhary31 LGTM

**Comment by ParthChaudhary31** [comment]
@semuelle ,
I understand your motives for keeping the Arbiter voting on-chain, The change is a suggestion that we believe would smoothen the onboarding process and encourage arbiters, however, we're perfectly fine with keeping it on-chain as well. If you can verify with the remaining team and give a final verdict on the changes, We'll be able to plan accordingly and start with the development.

**Review by semuelle — APPROVED** [review]
Thanks, @ParthChaudhary31. Sounds good to me. I will ping the rest of the committee again.

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks, @ParthChaudhary31. Sounds good to me. Could you integrate this into the document? I will ping the rest of the committee again.

**Comment by ParthChaudhary31** [comment]
@semuelle 
Sure, I've re-added the Arbiter's voting contract to milestone 1, We'll start working on it now and aim to finish this soon.

**Comment by ParthChaudhary31** [comment]
recheck

**Comment by ParthChaudhary31** [comment]
> recheck

recheck

File: applications/Security_Marketplace.md [hunk 1]
```diff
@@ -86,9 +86,10 @@ Here is an envisioned lifecycle of a project on our platform.
 19. The auditors will get platform Points along their profile based on their actions, for every:
     1 successful audit completion: 1XP point (Increases with more and more audits)
     1 unsuccessful audit: -3XP points (Adds up)
-20. The point system is designed to help patrons identify reputable members of the community as well as alert them of newcomers and bad elements. Once a user hits a -6XP rating, he cannot bid anymore
+20. The Arbiter's voting system will be hosted off chain to facilitate their transactions without any cost.
```
Comment (semuelle) [inline applications/Security_Marketplace.md:89]:
```suggestion

```

**Review by keeganquigley — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Pull request merged**